packages/common/src/utils/formatUtil.ts

```diff
@@ -129,7 +129,7 @@ export const pluralize = (
   count: number | null,
   suffix = 's',
   pluralizeAnyway = false
-) => `${message}${(count ?? 0) > 1 || pluralizeAnyway ? suffix : ''}`
+) => `${message}${(count ?? 0) !== 1 || pluralizeAnyway ? suffix : ''}`
```

Comment:
curious what the reasoning for this is? I would think we don't want to account for negatives?

Reply:
This covers the 0 case properly. We do want the suffix for 0 count.

0 reposts
1 repost
2+ reposts